doc/source/notebooks/manual/advanced/natural_gradients.ipynb

```diff
@@ -4,15 +4,881 @@
+    "\n",
+    "vgp_bernoulli = VGP(X, Y_binary, kern=make_matern_kernel(), likelihood=gpflow.likelihoods.Bernoulli())\n",
+    "vgp_bernoulli_natgrad = VGP(X, Y_binary, kern=make_matern_kernel(), likelihood=gpflow.likelihoods.Bernoulli())\n",
+    "\n",
+    "# ordinary gradients with Adam for VGP with Bernoulli likelihood\n",
+    "adam = AdamOptimizer(adam_learning_rate)\n",
+    "adam_for_vgp_bernoulli_tensor = adam.make_optimize_tensor(vgp_bernoulli)\n",
+    "\n",
+    "# NatGrads and Adam for VGP with Bernoulli likelihood\n",
+    "# Stop Adam from optimizing the variational parameters\n",
+    "vgp_bernoulli_natgrad.q_mu.trainable = False\n",
+    "vgp_bernoulli_natgrad.q_sqrt.trainable = False\n",
+    "\n",
+    "# Create the optimize_tensors for VGP with natural gradients\n",
+    "adam = AdamOptimizer(adam_learning_rate)\n",
+    "adam_for_vgp_bernoulli_natgrad_tensor = adam.make_optimize_tensor(vgp_bernoulli_natgrad)\n",
+    "\n",
+    "natgrad = NatGradOptimizer(gamma=.1)\n",
+    "variational_params = [(vgp_bernoulli_natgrad.q_mu, vgp_bernoulli_natgrad.q_sqrt)]\n",
+    "natgrad_tensor = natgrad.make_optimize_tensor(vgp_bernoulli_natgrad, var_list=variational_params)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 71,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Optimize vgp_bernoulli\n",
+    "for _ in range(notebook_niter(100)):\n",
+    "    session.run(adam_for_vgp_bernoulli_tensor)\n",
+    "\n",
+    "vgp_bernoulli.anchor(session)\n",
+    "\n",
+    "# Optimize vgp_bernoulli_natgrad\n",
+    "for _ in range(notebook_niter(100)):\n",
+    "    session.run(adam_for_vgp_bernoulli_natgrad_tensor)\n",
+    "    session.run(natgrad_tensor)\n",
+    "\n",
+    "vgp_bernoulli_natgrad.anchor(session)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "VGP likelihood after ordinary Adam optimization:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 72,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-146.1206"
+      ]
+     },
+     "execution_count": 72,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vgp_bernoulli.compute_log_likelihood()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "VGP likelihood after NatGrad + Adam optimization:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-143.9411"
+      ]
+     },
+     "execution_count": 73,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "vgp_bernoulli_natgrad.compute_log_likelihood()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also choose to run natural gradients in another parameterization.</br>\n",
```

Comment:
The `</br>` didn't render correctly.

Reply:
What do you mean?

Reply:
if I open the notebook file, Their is no line break, I can just see the `<br>`

Reply:
How do you open it? View file gives normal line break. I can't see anything.

Reply:
Chrome webbrowser. It looks fine now - maybe the page didn't load completely.